tools/pd-ctl/README.md

```diff
@@ -98,50 +98,60 @@ Usage:
   "replication": {
     "location-labels": "",
     "max-replicas": 3,
-    "strictly-match-label": "true"
+    "strictly-match-label": "false"
   },
   "schedule": {
-    "enable-one-way-merge": "false",
     "enable-cross-table-merge": "false",
-    "disable-location-replacement": "false",
-    "disable-make-up-replica": "false",
-    "disable-remove-down-replica": "false",
-    "disable-remove-extra-replica": "false",
-    "disable-replace-offline-replica": "false",
+    "enable-location-replacement": "true",
+    "enable-make-up-replica": "true",
+    "enable-one-way-merge": "false",
+    "enable-remove-down-replica": "true",
+    "enable-remove-extra-replica": "true",
+    "enable-replace-offline-replica": "true",
     "high-space-ratio": 0.6,
     "hot-region-cache-hits-threshold": 3,
-    "hot-region-schedule-limit": 2,
+    "hot-region-schedule-limit": 4,
```

Comment:
where do we update?

Reply:
Both of them are the default value, but we didn't update the doc when PD was merged.